loopchain/blockchain/blockchain.py

```diff
@@ -661,7 +667,8 @@ def confirm_prev_block(self, current_block: Block):
         #                    f"tx count({len(current_block.body.transactions)}), "
         #                    f"height({current_block.header.height})")
 
-        candidate_blocks = ObjectManager().channel_service.block_manager.candidate_blocks
+        block_manager: BlockManager = ObjectManager().channel_service.block_manager
```

Comment:
This type hint `BlockManager` at Line 37 is not valid on runtime.
If exceptions is not raised there, another importing `BlockManager` exists. (Maybe import *)

You should block_manager: 'BlockManager' if there is no other importing `BlockManager`.